.changeset/bright-kings-relate.md

```diff
@@ -0,0 +1,23 @@
+---
+'@backstage/plugin-catalog-react': minor
+'@backstage/create-app': patch
+'@backstage/plugin-catalog': minor
```

Comment:
When the deprecated code is eventually removed, it'll definitely be `major`, but perhaps `patch` (instead of `minor`) on `plugin-catalog` is more appropriate for now, since the API surface area isn't really changing.